sh2lib/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+version: "0.1.0"
```

Comment:
Do we really want a version with a leading zero?  According to semver 2.0 https://semver.org/#spec-item-4 
Public API should be considered unstable at this point. 
If we are going to include it to IDF, I would suggest to release this with 
`version: 1.0.0` 

and add dependency to the IDF example as 
`espressif/sh2lib: "~1.0.0"` (>=1.0.0,<2.0.0)

Reply:
Good point. Agree that we should start with version `1.0.0.` here. I have fixed this.